packages/server/utils/ensureJiraDimensionField.ts

```diff
@@ -64,6 +64,7 @@ const ensureJiraDimensionField = async (
         jiraDimensionFields: team('jiraDimensionFields')
           .default([])
           .union(fieldsToAdd)
```

Comment:
ohhhhh! this is probably the culprit.
i think rethinkdb's union method is non-deterministic

Reply:
That's right! [Doc](https://rethinkdb.com/api/javascript/union): 
```
results will be mixed together; this is the fastest setting, but ordering of elements is not guaranteed. (This is the default.)
```